src/python/pants/backend/codegen/protobuf/lint/buf/skip_field.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.backend.codegen.protobuf.target_types import (
+    ProtobufSourcesGeneratorTarget,
+    ProtobufSourceTarget,
+)
+from pants.engine.target import BoolField
+
+
+class SkipBufField(BoolField):
+    alias = "skip_buf_lint"
```

Comment:
Relates to my question on the subsystem name, whether this should be `skip_buf` vs `skip_buf_lint`. We don't use `_lint` with anything else: `skip_black`, `skip_shfmt`, etc. But also those tools are dedicated linters/formatters, unlike Buf, so it seems acceptable and maybe desirable to diverge. Wdyt?